### PR TITLE
Change "Requestor Information" to "Personal Information" and make requirements clearer in VBA flow

### PR DESCRIPTION
**Pull request by luacmartins**
### Details
This PR changes `Requestor information` to `Personal information`, as well as updates the URL to the step and adds a clarifying line below the header text.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5081

### Tests
1. Login to an account with a Workspace, but without a bank account setup.
2. Open the VBA flow by clicking on `Settings > Select Workspace > Get Started`
3. Fill the form until you get to the `Personal information` step. You can add [these credentials](https://stackoverflow.com/c/expensify/questions/342/525#525) if needed.
4. Verify that it looks like the screenshots below.

### QA Steps
Steps above.

### Tested On

- [X] Web
- [X] Mobile Web
- [X] Desktop
- [X] iOS
- [X] Android

#### Web
web

#### Mobile Web
mWeb

#### Desktop
desktop

#### iOS
ios

#### Android
android

### CODE REVIEW

**Comment by shawnborton** [comment]
What do you think about adding just a little more separation between the blue links and the line above? Maybe try adding `mt1` or `mt2`?

**Comment by luacmartins** [comment]
This is how it looks with `mt1`:
Screen Shot 2021-09-08 at 6 07 12 PM

**Comment by shawnborton** [comment]
Nice, I like that better! What are your thoughts?

**Comment by luacmartins** [comment]
It does look better. I'll make the change and update the images. Thanks @shawnborton!

File: src/pages/ReimbursementAccount/RequestorStep.js [hunk 1]
```diff
@@ -135,6 +135,24 @@ class RequestorStep extends React.Component {
                     <>
                         <ScrollView style={[styles.flex1, styles.w100]}>
                             <View style={[styles.p4]}>
+                                <Text>{this.props.translate('requestorStep.subtitle')}</Text>
+                                <View style={[styles.mb5, styles.mt1, styles.dFlex, styles.flexRow]}>
+                                    <TextLink
+                                        style={[styles.textMicro]}
+                                        // eslint-disable-next-line max-len
```
Comment (Julesssss) [inline src/pages/ReimbursementAccount/RequestorStep.js:142]: Do we still need this line? I thought that we increased the max line count.

File: src/pages/ReimbursementAccount/RequestorStep.js [hunk 2]
```diff
@@ -135,6 +135,24 @@ class RequestorStep extends React.Component {
...
+                                        href="https://community.expensify.com/discussion/6983/faq-why-do-i-need-to-provide-personal-documentation-when-setting-up-updating-my-bank-account"
+                                    >
+                                        {`${this.props.translate('requestorStep.learnMore')}`}
+                                    </TextLink>
+                                    <Text style={[styles.textMicroSupporting]}>{' | '}</Text>
+                                    <TextLink
+                                        style={[styles.textMicro, styles.textLink]}
+                                        // eslint-disable-next-line max-len
```
Comment (Julesssss) [inline src/pages/ReimbursementAccount/RequestorStep.js:150]: And here

**Review by Julesssss — APPROVED** [review]
Looks good other than the eslint comments, but let's not block N6 on this.

**Pull request merged**